cypress/integration/core-opensearch-dashboards/opensearch-dashboards/mds_dashboard_sanity_test_spec.js

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  CommonUI,
+  MiscUtils,
+} from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const commonUI = new CommonUI(cy);
+const miscUtils = new MiscUtils(cy);
+const baseURL = new URL(Cypress.config().baseUrl);
+// remove trailing slash
+const path = baseURL.pathname.replace(/\/$/, '');
+
+const disableLocalCluster = !!Cypress.env('DISABLE_LOCAL_CLUSTER');
+
+if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
+  describe('dashboard data source sample data validation', () => {
+    before(() => {});
+
+    after(() => {});
+
+    describe('checking home page', () => {
+      before(() => {
+        // Go to the home page
+        miscUtils.visitPage('app/home#');
+        cy.window().then((win) =>
+          win.localStorage.setItem('home:welcome:show', false)
```

Comment:
Hi. Could you help me to understand why we add this home:welcome:show in localStorage?

Reply:
@zhyuanqi 
it's for hiding this welcome page
![image](https://github.com/opensearch-project/opensearch-dashboards-functional-test/assets/32652829/9bbca877-d2c1-4857-af37-4d33c0681988)